engine.py

```diff
@@ -442,6 +442,21 @@ def __init_qt_app(self):
         if os.path.exists(plugins_dir):
             QtCore.QCoreApplication.addLibraryPath(plugins_dir)
 
+        if QtCore.qVersion()[0] == "5":
```

Comment:
You could also use the engine property to check if running with Qt5. The `__has_qt5` engine attribute will get initialized before calling the engine `pre_app_init` method which call this function.
```suggestion
        if self.has_qt5:
```